.github/CONTRIBUTING.md

```diff
@@ -124,8 +124,9 @@ In addition, linters can help to enforce coding standards, find unused variables
 
 **HospitalRun** uses the following linters:
 
-1. [ESLint](http://eslint.org/) for ECMAScript/JavaScript. You can find the ESLint User guide [here](http://eslint.org/docs/user-guide/).
+1. [ESLint](http://eslint.org/) for ECMAScript/JavaScript. You can find the ESLint User guide [here](http://eslint.org/docs/user-guide/). In addition to the default set of rules ESLint is setup to use eslint-plugin-ember-suave you can find more information about that [here](https://github.com/DockYard/eslint-plugin-ember-suave).
```

Comment:
We need a link here to Dockyard's Ember.js style guide:
https://github.com/DockYard/styleguides/blob/master/engineering/ember.md

This has more helpful information than the  eslint-plugin-ember-suave page.